compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/LangLibrary.java

```diff
@@ -125,6 +126,9 @@ public static LangLibrary getInstance(CompilerContext context) {
     private void populateMethodList(List<FunctionSymbol> list, Map<String, BInvokableSymbol> langLib, BType type,
                                     BType boundTypeParam) {
         for (BInvokableSymbol symbol : langLib.values()) {
+            if (symbol.origin == SymbolOrigin.VIRTUAL) {
```

Comment:
How is it that we need this change only now? Wasn't it an issue with the closure map (for normal closures)?

Reply:
With the new changes, we didn't need this check.